data/darktableconfig.xml.in

```diff
@@ -1222,6 +1222,13 @@
     <shortdescription>scroll to darkroom modules when expanded/collapsed</shortdescription>
     <longdescription>when this option is enabled then darktable will try to scroll the module to the top of the visible list</longdescription>
   </dtconfig>
+  <dtconfig prefs="gui" section="darkroom">
```

Comment:
this can't be in the darkroom section has it also works on lighttable. so please do as I proposed in my previous review.

Reply:
I only changed the section from darkroom to miscellaneous as proposed and now 'all checks have failed'.
I don't know why this has failed...